job.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       hostPID: true
       containers:
         - name: kube-bench
-          image: aquasec/kube-bench:0.6.3
+          image: aquasec/kube-bench:v0.6.6
```

Comment:
How about changing it to `latest` ?

Reply:
Not sure if `latest` will be always stable. Also best practice is to use specific tag instead of latest.